py/picca/fitter2/chi2.py

```diff
@@ -274,16 +285,22 @@ def fastMC(self):
 
     def minos(self):
         if not hasattr(self,"minos_para"): return
-
         sigma = self.minos_para['sigma']
+        cl=scipy.stats.norm.cdf(sigma, loc=0, scale=1)-scipy.stats.norm.cdf(-sigma, loc=0, scale=1)    
```

Comment:
Could the difference in the results be originated here? Or is this not entering the tests?

Reply:
nope, this is part of fitter2 which by now is giving the correct result. The change is because nomenclature in minos calls went from N-sigma to giving minos the confidence range you want, the `cl=...` here is just for doing the conversion.

It might be that we get different results for an N-d minos fit with N>1, this is not part of the test case and I'm not certain if the sigma used for previous versions was supposed to mean 1σ-confidence range of a 1-d Gaussian (i.e. 68%) or 1σ-confidence range of a N-d Gaussian (which would be <68%). In the implementation so far I assume the former, but it might be the latter...

Reply:
ok so then I guess the comments might be a bit misleading?